coccinelle/.gitignore

```diff
@@ -0,0 +1 @@
+/.coccinelle-cache
```

Comment:
Add in the top level .gitignore with fuzzy match?

Reply:
I am not familiar about 'fuzzy match', but docs/.gitignore have similar entry for caching.